docs/source/quick_start/install.rst

```diff
@@ -0,0 +1,218 @@
+
+Then install MCDR using pipx:
+
+.. code-block:: bash
+
+    $ pipx install mcdreforged
+
+When a new version of MCDR available, you may take the upgrade by:
+
+.. code-block:: bash
+
+    $ pipx upgrade mcdreforged
+
+.. asciinema:: resources/pipx.cast
+    :rows: 8
+
+|
+
+.. note::
+
+    In this way, MCDR will be installed in an isolated environment. Instead of ``pip install <package_name>``, Python packages required by MCDR plugins should be installed by:
+
+    * ``pipx inject mcdreforged <package_name>``
+    * or ``pipx inject mcdreforged -r requirements.txt``
+    
+    More conveniently, use the :ref:`\!!MCDR plg command <command/mcdr:Plugin management>` to install plugins with their dependencies
+
+
+Using virtual environment
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For the most native, but more complicated option, you can create a virtual environment and install MCDR in it
+
+Create a virtual environment by:
+
+.. code-block:: bash
+
+    $ python3 -m venv <venv directory>
+
+``venv`` for example:
+
+.. code-block:: bash
+
+    $ python3 -m venv venv
+
+Activate it by:
+
++----------+------------+-----------------------------------------+
+| Platform | Shell      | Command to activate virtual environment |
++==========+============+=========================================+
+|  POSIX   | bash/zsh   | $ source venv/bin/activate              |
++          +------------+-----------------------------------------+
+|          | fish       | $ source venv/bin/activate.fish         |
++          +------------+-----------------------------------------+
+|          | csh/tcsh   | $ source venv/bin/activate.csh          |
++          +------------+-----------------------------------------+
+|          | PowerShell | $ venv/bin/Activate.ps1                 |
++----------+------------+-----------------------------------------+
+| Windows  | cmd.exe    | > venv\\Scripts\\activate.bat           |
++          +------------+-----------------------------------------+
+|          | PowerShell | PS > venv\\Scripts\\Activate.ps1        |
++----------+------------+-----------------------------------------+
+
+Then, install MCDR using pip:
+
+.. code-block:: bash
+
+    (venv) $ pip install mcdreforged
+
+When a new version of MCDR available, you may upgrade MCDR by:
+
+.. code-block:: bash
+
+    (venv) $ pip install mcdreforged -U
+
+An animated demo with bash:
+
+.. asciinema:: resources/venv.cast
+    :rows: 10
+
+|
+
+.. note::
+
+    In this way, you must activate the virtual environment every time you want to use MCDR, or install packages for MCDR plugins
+
+.. seealso ::
+
+    Python Doc: `How venvs work <https://docs.python.org/3/library/venv.html#how-venvs-work>`__
+
+Using Docker
+~~~~~~~~~~~~
+
+MCDR also provides Docker images as an option. See :doc:`/docker` for more details
+
+Specify MCDR version by specify the tag of Docker image. If you use ``latest``, MCDR should always being up to date
```

Comment:
You need to pull the image to keep the local ``latest`` image up-to-date

Reply:
seems that github copilot is still not versatile enough